### ISSUE TITLE
Add Android homescreen shortcut to do action in osmand

### ISSUE THREAD

**Issue by CorentinLemaitre** [osmandapp/Osmand#7037]
Hello,

I am using osmand for android. My version is 3.3.8

It is not an issue, it is functionnality nice to have. 

I would like that the action that can be set in the quick action could be used as shortcut outside the app. I mean the kind of shortcut that you can add to your android home screen or assign to button or specific swipe. 

My goal is to create osm notes or audio note or POI even when i haven't start the app.
By the way : A huge thanks to all the developpers and team for osmand !

**Comment by vshcherb** [comment]
Yes it is a nice to have. As of today please use Quick Action and create them to do certain things

**Comment by jkufner** [comment]
The use of quick actions to create the desktop (homescreen) shortcuts is probably reasonable approach, but then we need to add an option to use favorites (and other locations) as a parameter of a quick action. For example, a quick action that starts a navigation to a given place.

**Comment by Jacekalex** [comment]
You can work around this issue differently:
install Shortcut maker or another program:
create a http link with the content of the type, for example:
http://osmand.net/go?lat=51.4404&lon=4.3294&z=16
and place it on the homescreen.
After that, click on this shortcut, and when the system asks in which application to open such links, 
select the default osmand +.

Cheers

**Comment by jkufner** [comment]
@Jacekalex That is a good point and could work in some use cases, but it won't start the navigation with a single click. Osmand starts for some time and thus the interaction in the begining is time consuming, or at least distracting.

Imagine this scenario: I enter a car, put my phone in the holder and click a shortcut on the home screen. Osmand starts, loads for some time, and then starts navigating to the desired location. In mean time I can start the car, remove snow and ice, get out of a tight parking spot, ... Osmand has plenty of time to start navigating before I get on the main road. The point is that I do not have to pay any further attention to it after clicking the shortcut.

**Comment by Jacekalex** [comment]
**@jkufner**
Therefore, it is necessary to request that OSMAND support http links in the following format:
https://osmand.net/go?from={CURRENT_LOCATION?}&to={DEST_LOCATION}&method=+car,+railway,+sea_ship,+etc... 
that is, it allowed to pass multiple parameters in a GET query, which will greatly simplify the program's interaction, e.g. in a car or a bicycle.

Cheers

**Comment by CorentinLemaitre** [comment]
I have seen the discussion going on about this issue. 

I think in the message of @jkufner, the goal is not the same (routing or showing specific place). These functions are the core functions for every map application, and it is possible using deeplink as @Jacekalex mentioned.

I think i forget to share information about what quick action in OSMand can do. When you use quick action your location is taken as the default value. 

Here is the list of what quick action can do : 
- Create a favourite point in the folder defined before
- Add a audio or video or photo note
- Add OSM note or create point in OSM
- And also modify map settings or navigation settings

**Comment by dmpr0** [comment]
Duplicate https://github.com/osmandapp/OsmAnd/issues/6555
